rpcs3/rpcs3qt/save_manager_dialog.cpp

```diff
@@ -100,47 +107,99 @@ void save_manager_dialog::Init(std::string dir)
 	m_list->setSelectionMode(QAbstractItemView::SelectionMode::ExtendedSelection);
 	m_list->setSelectionBehavior(QAbstractItemView::SelectRows);
 	m_list->setContextMenuPolicy(Qt::CustomContextMenu);
-	m_list->setColumnCount(4);
-	m_list->setHorizontalHeaderLabels(QStringList() << tr("Title") << tr("Subtitle") << tr("Save ID") << tr("Entry Notes"));
-
-	QPushButton* push_remove_entries = new QPushButton(tr("Delete Selection"), this);
-
+	m_list->setColumnCount(5);
+	m_list->setHorizontalHeaderLabels(QStringList() << tr("Icon") << tr("Title & Subtitle") << tr("Last Modified") << tr("Save ID") << tr("Notes"));
+	m_list->horizontalHeader()->setSectionResizeMode(0, QHeaderView::Fixed);
+	m_list->horizontalHeader()->setStretchLastSection(true);
+
+	// Bottom bar
+	int icon_size = m_gui_settings->GetValue(gui::sd_icon_size).toInt();
+	m_icon_size = QSize(icon_size, icon_size);
+	QLabel* label_icon_size = new QLabel("Icon size:", this);
+	QSlider* slider_icon_size = new QSlider(Qt::Horizontal, this);
+	slider_icon_size->setMinimum(60);
+	slider_icon_size->setMaximum(225);
+	slider_icon_size->setValue(m_icon_size.height());
 	QPushButton* push_close = new QPushButton(tr("&Close"), this);
 	push_close->setAutoDefault(true);
 
-	// Button Layout
+	// Details
+	m_details_icon = new QLabel(this);
+	m_details_icon->setMinimumSize(320, 176);
```

Comment:
Just wondering why you didn't just use a new form file.
Most of these settings can be done with simple mouse clicks

Reply:
I just started with small tweaks to the current UI and it grew from there line by line. Besides I'm used to creating interfaces with code rather than external editor.

Reply:
That's fine. As I said, I was just curious.